packages/http/libraries/core/README.md

```diff
@@ -0,0 +1,6 @@
+[![Test coverage](https://codecov.io/gh/inversify/monorepo/branch/main/graph/badge.svg?flag=%40inversifyjs%2Fcommon)](https://codecov.io/gh/inversify/monorepo/branch/main/graph/badge.svg?flag=%40inversifyjs%2Fcommon)
+[![npm version](https://img.shields.io/github/package-json/v/inversify/monorepo?filename=packages%2Fcontainer%2Flibraries%2Fcommon%2Fpackage.json&style=plastic)](https://www.npmjs.com/package/@inversifyjs/common)
```

Comment:
Let's update these links to point to the right libraries

Reply:
Fixed